packages/Calendar/Microsoft.Bot.Components.Calendar.csproj

```diff
@@ -6,11 +6,10 @@
     <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);CopyProjectReferencesToPackage</TargetsForTfmSpecificBuildOutput>
   </PropertyGroup>
 
-  <!-- Nuget packaging information -->
   <PropertyGroup>
-    <PackageId>Microsoft.Bot.Component.CalendarSkill</PackageId>
-	<VersionPrefix>1.0.0</VersionPrefix>
-	<VersionSuffix>alpha.1</VersionSuffix>
+    <PackageId>Microsoft.Bot.Components.Calendar</PackageId>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionSuffix>preview1</VersionSuffix>
```

Comment:
If this is still private should remain "alpha.1"
If it should be public then rename "preview.1"